rock_paper_scissors.py

```diff
@@ -0,0 +1,61 @@
+#!/bin/python3
+print('''Let\'s play a game.
+''')
+def playgame():
+  player = input('Rock (1), paper (2) or scissors (3)?')
+  player = int(player)
```

Comment:
I would put both of these on one line, but I understand if you want to keep them separate for teaching purposes.